rust/test_service_elections/src/lib.rs

```diff
@@ -71,6 +71,10 @@ mod service {
         }
     }
 
+    fn get_current_time() -> TimePointSec {
+        transaction_sys::Wrapper::call().headBlockTime()
```

Comment:
headBlockTime is marked `TODO: remove`. Use `currentBlock()`, which is more suitable than `headBlock()` for most cases.